packages/storefront/src/store.ts

```diff
@@ -43,8 +45,12 @@ export default new Vuex.Store({
     setSelectedFramework(state: State, payload: Framework): void {
       state.selectedFramework = payload;
     },
-    setTheme(state: State, payload: Theme): void {
-      state.theme = payload;
+    setPlaygroundTheme(state: State, payload: Exclude<Theme, 'auto'>): void {
+      localStorage.setItem('playgroundTheme', payload);
```

Comment:
Why only persist one of the themes and not both?
Perhaps "syncing" it to state could also be skipped when getters just return it from localStorage.

Reply:
Because the `platformTheme` works differently than `playgroundTheme`. `platformTheme` by default uses the auto dark mode and depending on the OS settings of the user the playgroundTheme may be shown in dark or light. Depending on that individual case the option to cycle to a static theme light or dark is chosen.

Imagine I've set my OS settings to theme=auto and I visit the page at night. Then I'll see the page with dark theme. Then I click on cycle theme. Then I would see the page in light theme. If I then would store this setting in local storage (theme=light), exit the page and visit it again on the next day. Then I would see a light page, and if I would then click on cycle theme, I would use theme=auto as next step which again would produce a light page. I hope it's somehow understandable. :)